frontend/lib/src/hostComm/HostCommunicationManager.test.tsx

```diff
@@ -98,11 +98,16 @@ describe("HostCommunicationManager messaging", () => {
     expect(hostCommunicationMgr.allowedOrigins).toEqual([
       "https://devel.streamlit.test",
     ])
-    expect(openCommFunc).toHaveBeenCalledWith()
+    expect(openCommFunc).toHaveBeenCalled()
   })
 
   it("host should receive a GUEST_READY message", () => {
-    expect(sendMessageToHostFunc).toHaveBeenCalledWith({ type: "GUEST_READY" })
+    expect(sendMessageToHostFunc).toHaveBeenCalled()
+
+    const guestReadyMessage = sendMessageToHostFunc.mock.calls[0][0]
+    expect(guestReadyMessage).toHaveProperty("type", "GUEST_READY")
+    expect(guestReadyMessage).toHaveProperty("streamlitExecutionStartedAt")
```

Comment:
nit: do you have access to the variable `window.ReactLoad` here and can also compare that the field `streamlitExecutionStartedAt === window.ReactLoad` or is it not set in the unit test? If it is, it would harden the test to include the check as you then also have covered the value & usage of `window.ReactLoad`.

Reply:
Unfortunately it is not accessible/set in the unit test (`streamlitExecutionStartedAt` is undefined)